### PR TITLE
add the focus text editor theme

### PR DESCRIPTION
**Pull request by smezzy**
perfect for gorgeous looks, can push asap
image

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -518,6 +518,17 @@
          ],
          "type" : "color",
          "version" : "0.1"
+      },
+      {
+         "id" : "focus-theme",
```
Comment (Guldoman) [inline manifest.json]: Might be a good idea to just call it `focus`. Same for the file name.
What do you think?

**Review by Guldoman — COMMENTED** [review]
Looks good!
Could you sort the entry in the manifest?
I just noticed that `bearded-theme-vivid-black` is unsorted too agh.

File: manifest.json (hunk 1 above)
Comment (smezzy) [reply manifest.json]: i like it i prefer focus i was going to call it focus initially but i  thought it was more professional ™

**Comment by smezzy** [comment]
ive sorted the plugins including the bearded theme, i saw it last on the list and i just followed its footsteps xD sorry

File: manifest.json [hunk 2]
```diff
@@ -508,16 +530,5 @@
          "type" : "color",
          "version" : "0.1"
       },
```
Comment (Guldoman) [inline manifest.json]:
```suggestion
      }
```
because JSON is picky.

File: colors/focus.lua [hunk 3]
```diff
@@ -0,0 +1,34 @@
+-- color theme from the focus text editor
+
+local style = require "core.style"
+local common = require "core.common"
+
+style.caret_width = 2
```
Comment (Guldoman) [inline colors/focus.lua]: Is this needed?
Might be a good idea to either remove it or do:
```suggestion
style.caret_width = common.round(2 * SCALE)
```
but this is the default anyways, so might as well remove it.

**Comment by Guldoman** [comment]
> ive sorted the plugins including the bearded theme, i saw it last on the list and i just followed its footsteps xD sorry

It's my bad, I forgot to check when it was PRd.

**Comment by Guldoman** [comment]
Looks good! Thanks!

**Pull request merged**